libvlc.yml

```diff
@@ -0,0 +1,347 @@
+- name: libmodplug
+  rm-configure: true
+  sources:
+  - type: archive
+    url: http://http.debian.net/debian/pool/main/libm/libmodplug/libmodplug_0.8.9.0.orig.tar.gz
+    sha256: 457ca5a6c179656d66c01505c0d95fafaead4329b9dbaa0f997d00a3508ad9de
+  - type: script
+    commands:
+    - autoreconf -fiv
+    dest-filename: autogen.sh
+- name: x264
+  cleanup:
+  - "/bin"
+  config-opts:
+  - "--enable-pic"
+  - "--enable-shared"
+  sources:
+  - type: git
+    url: https://code.videolan.org/videolan/x264.git
+    branch: d198931a63049db1f2c92d96c34904c69fde8117
+- name: x265
+  cleanup:
+  - "/bin"
+  builddir: true
+  subdir: source
+  buildsystem: cmake-ninja
+  config-opts:
+  - "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+  - "-DBUILD_SHARED_LIBS=ON"
+  - "-DCMAKE_POSITION_INDEPENDENT_CODE=ON"
+  sources:
+  - type: archive
+    url: https://ftp.osuosl.org/pub/blfs/conglomeration/x265/x265_3.4.tar.gz
+    sha256: c2047f23a6b729e5c70280d23223cb61b57bfe4ad4e8f1471eeee2a61d148672
+- name: dav1d
+  cleanup:
+  - "/bin"
+  buildsystem: meson
+  sources:
+  - type: archive
+    url: https://get.videolan.org/dav1d/0.9.0/dav1d-0.9.0.tar.xz
+    sha256: cfae88e8067c9b2e5b96d95a7a00155c353376fe9b992a96b4336e0eab19f9f6
+- name: ffmpeg
+  cleanup:
+  - "/bin"
+  - "/share"
+  config-opts:
+  - "--enable-pic"
+  - "--enable-shared"
+  - "--disable-doc"
+  - "--disable-static"
+  - "--enable-gpl"
+  - "--enable-libx264"
+  - "--enable-libx265"
+  - "--enable-libvpx"
+  - "--enable-libmp3lame"
+  - "--enable-libvorbis"
+  - "--enable-libopus"
+  sources:
+  - type: archive
+    url: http://ffmpeg.org/releases/ffmpeg-4.4.tar.xz
+    sha256: 06b10a183ce5371f915c6bb15b7b1fffbe046e8275099c96affc29e17645d909
+- name: fluidlite
+  buildsystem: cmake-ninja
+  config-opts:
+  - "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+  - "-DBUILD_SHARED_LIBS=ON"
+  - "-DCMAKE_POSITION_INDEPENDENT_CODE=ON"
+  sources:
+  - type: git
+    url: https://github.com/divideconcept/FluidLite.git
+    branch: fdd05bad03cdb24d1f78b5fe3453842890c1b0e8
+- name: libplacebo
+  cleanup:
+  - "/bin"
+  buildsystem: meson
+  sources:
+  - type: archive
+    url: https://github.com/haasn/libplacebo/archive/refs/tags/v3.120.3.tar.gz
+    sha256: aeea8edb38af8034fa592f7c028aa612823707087db32e11908ba8ce72a9f419
+- name: protobuf
+  cleanup:
+  - "/bin"
+  config-opts:
+  - "--disable-static"
+  sources:
+  - type: archive
+    url: https://github.com/google/protobuf/archive/v3.5.1/protobuf-3.5.1.tar.gz
+    sha256: 826425182ee43990731217b917c5c3ea7190cfda141af4869e6d4ad9085a740f
+  - type: archive
+    url: https://github.com/google/googlemock/archive/release-1.7.0.tar.gz#/googlemock-1.7.0.tar.gz
+    sha256: 3f20b6acb37e5a98e8c4518165711e3e35d47deb6cdb5a4dd4566563b5efd232
+    dest: gmock
+  - type: archive
+    url: https://github.com/google/googletest/archive/release-1.7.0.tar.gz#/googletest-1.7.0.tar.gz
+    sha256: f73a6546fdf9fce9ff93a5015e0333a8af3062a152a9ad6bcb772c96687016cc
+    dest: gmock/gtest
```

Comment:
other packages build without these. Since test aren't run, maybe it's worth trying without. Would save on maintenance.

Reply:
Thanks for the suggestion, I will try to remove it.